.circleci/docker/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM ubuntu:16.04 AS builder
+
+RUN apt-get update --fix-missing
+RUN apt-get install -y --fix-missing \
+  openjdk-8-jdk-headless \
+  ant \
+  build-essential \
+  cmake \
+  python \
+  git \
+  moreutils
+
+WORKDIR /opt/voltdb
+
+COPY . .
+
+RUN ant clean
+RUN ant dist | ts '[%Y-%m-%d %H:%M:%S]'
+RUN cp obj/release/voltdb-community-*.tar.gz obj/release/voltdb-community.tar.gz
+
+
+
+FROM ubuntu:16.04
+
+RUN apt-get update --fix-missing
+RUN apt-get install -y --fix-missing \
+  python \
+  openjdk-8-jre-headless
+
+ENV VOLTDB_DIST=/opt/voltdb
+ENV PATH=$PATH:$VOLTDB_DIST/bin
+
+COPY --from=builder /opt/voltdb/obj/release/voltdb-community.tar.gz voltdb-community.tar.gz
+
+RUN tar -zxf voltdb-community.tar.gz
+RUN mv voltdb-community-* voltdb-community
+RUN mkdir -p ${VOLTDB_DIST}
+RUN cp -r voltdb-community/* $VOLTDB_DIST
```

Comment:
Not a blocker, but you may potentially be able replace this block of commands with a single, adjusted `tar` command (see https://serverfault.com/a/330131):

```docker
RUN tar -zxf voltdb-community.tar.gz --strip-components=1 -C $VOLTDB_DIST
```

I think `--strip-components=1` will suffice, but you may need to play with the number.

Reply:
Follow up addressed here: https://github.com/mode/voltdb/pull/10/files
Thanks for the suggestion!